jni/include/com/amazonaws/kinesis/video/producer/jni/ClientJVMContext.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <jni.h>
+
+struct ClientJVMContext {
+    JavaVM* jvm;
+    jobject javaObjectRef;          // Pointer to NativeKinesisVideoProducerJni java object
```

Comment:
javaObjectglobalRef ?
```
if (NULL == (mJVMContext.javaObjectRef = env->NewGlobalRef(thiz))) {
        DLOGE("Failed to create a global reference for the JNI object.");
        return FALSE;
    } 
```

Reply:
I didn't want to include the word "global" to make it less confusing